src/app/saga.test.ts

```diff
@@ -547,61 +548,11 @@ describe('runCentralPhoneVerificationMigration', () => {
   })
 })
 
-describe('appInit', () => {
-  beforeEach(() => {
-    jest.clearAllMocks()
-  })
-
-  const defaultProviders: (EffectProviders | StaticProvider)[] = [
-    [select(allowOtaTranslationsSelector), true],
-    [select(otaTranslationsAppVersionSelector), '1'],
-    [select(currentLanguageSelector), 'nl-NL'],
-    [select(sentryNetworkErrorsSelector), 'network error'],
-  ]
-
-  it('should initialise the correct components, with the stored language', async () => {
-    await expectSaga(appInit)
-      .provide(defaultProviders)
-      .put(setSupportedBiometryType(BIOMETRY_TYPE.TOUCH_ID))
-      .run()
-
-    expect(initializeSentry).toHaveBeenCalledTimes(1)
-    expect(ValoraAnalytics.init).toHaveBeenCalledTimes(1)
-    expect(initI18n).toHaveBeenCalledWith('nl-NL', true, '1')
-  })
-
-  it('should initialise with the best language', async () => {
-    jest
-      .spyOn(RNLocalize, 'findBestAvailableLanguage')
-      .mockReturnValue({ languageTag: 'de-DE', isRTL: true })
-
-    await expectSaga(appInit)
-      .provide([[select(currentLanguageSelector), null], ...defaultProviders])
-      .put(setSupportedBiometryType(BIOMETRY_TYPE.TOUCH_ID))
-      .run()
-
-    expect(initializeSentry).toHaveBeenCalledTimes(1)
-    expect(ValoraAnalytics.init).toHaveBeenCalledTimes(1)
-    expect(initI18n).toHaveBeenCalledWith('de-DE', true, '1')
-  })
-
-  it('should initialise with the app fallback language', async () => {
-    jest.spyOn(RNLocalize, 'findBestAvailableLanguage').mockReturnValue(undefined)
-
-    await expectSaga(appInit)
-      .provide([[select(currentLanguageSelector), null], ...defaultProviders])
-      .put(setSupportedBiometryType(BIOMETRY_TYPE.TOUCH_ID))
-      .run()
-
-    expect(initializeSentry).toHaveBeenCalledTimes(1)
-    expect(ValoraAnalytics.init).toHaveBeenCalledTimes(1)
-    expect(initI18n).toHaveBeenCalledWith('en-US', true, '1')
-  })
-})
-
+// Keep this test before the app init test otherwise the error spec will fail
```

Comment:
this is not okay 🙈 we should never have to place unit tests in a specific order. if this is a requirement, it means that there are interdependencies between tests which is a bad thing for a unit test! in this case i think that we have not cleared up the mocks from the app init tests, and that has caused the next tests to fail...